package.json

```diff
@@ -34,14 +34,17 @@
     "lint:editorconfig": "eclint check git-open* readme* .travis.yml",
     "lint:package": "pjv --recommendations --warnings",
     "lint:readme": "node ./node_modules/markdownlint/lib/markdownlint.js --config markdownlint.json README.md",
-    "test": "npm run unit && npm run lint:package && npm run lint:readme && npm run lint:editorconfig",
+    "lint:man": "node ./node_modules/markdownlint/lib/markdownlint.js --config markdownlint.json git-open.1.md",
+    "man": "marked-man --version \"git-open $npm_package_version\" --manual \"Git manual\" --section 1 git-open.1.md > git-open.1",
+    "test": "npm run unit && npm run lint:package && npm run lint:man && npm run lint:readme && npm run lint:editorconfig",
     "unit": "bats test/",
     "watch": "find . -maxdepth 2 -iname '*bats' -o -iname 'git-open' | entr bats test/"
   },
   "dependencies": {},
```

Comment:
As @4U6U57 said, you can support man automatically but using "https://docs.npmjs.com/files/package.json#man".

Reply:
These lines are to validate the markdown and build the man-page from the markdown, not to install. I've added the line you both mentioned.